source/code/plugins/in_npmd_server.rb

```diff
@@ -149,14 +152,15 @@ def log_error(msg, depth=0)
         end
 
         def kill_all_agent_instances
-            _cmd = "#{CMD_ENUMERATE_AGENT_PROCESSES}".chomp + " " + @binary_presence_test_string.chomp
+            _cmd = "#{CMD_ENUMERATE_PROCESSES_PREFIX}".chomp + " " + @binary_presence_test_string.chomp
             _resultStr = `#{_cmd}`
             return if _resultStr.nil?
             _lines = _resultStr.split("\n")
             _lines.each do |line|
                 if line.include?@binary_presence_test_string
-                    _userName = line.split()[0]
-                    _staleId = line.split()[1]
+                    _words = line.split()
```

Comment:
Requested change by @robbiezhang  in https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/187